### ISSUE TITLE
[Suggestion - Execute SQL tab] Access history via arrows

### ISSUE THREAD

**Issue by chrisjlocke** [sqlitebrowser/dbhub.io#194]
![image](https://user-images.githubusercontent.com/7872218/235347190-e2fd4829-4a47-469a-a1f2-68d1d56bb0c5.png)

Accidentally mistyped, so allowing accessing history via arrow keys would be helpful. :)

**Comment by MKleusberg** [comment]
Yep, fully agree :+1: And shouldn't be too hard to implement. The only bit I'm worried about is editing multi-line statements. You might want to go to the beginning so you hit arrow up a couple of times just to see your statement being replaced by the previous one. Not sure what's the best approach there...

**Comment by chrisjlocke** [comment]
Yes, the command prompt in Windows does that .. can be annoying! ;)  Think most people are familiar with this though .. maybe ..

**Comment by MKleusberg** [comment]
Just noticed that at least here on my Linux system you can recover the lost statement by pressing arrow down.

**Comment by justinclift** [comment]
Yeah, in the default bash setup (on Ubuntu 20.04), it seems to keep the "currently being edited" line in memory so it can be selected again for further editing by hitting the down arrow a few times.

**Comment by chrisjlocke** [comment]
> you can recover the lost statement by pressing arrow down

Which browser is that in?  In Firefox, that doesn't work for me (in Kubuntu).  Down arrow simply goes to the end of the line.
